src/main/java/ballsorting/Deadline.java

```diff
@@ -0,0 +1,38 @@
+package ballsorting;
```

Comment:
I think there should be a space between package and import statements? But am not sure whether it's included in our coding standards

Reply:
yep, there should be a space between package and import